themes/socialbase/components/01-base/tables/_tables.scss

```diff
@@ -48,7 +48,7 @@ th {
 
   // Bottom align for column headings
   thead th {
-    text-transform: uppercase;
+    //text-transform: uppercase;
```

Comment:
@Crazylord can you decide this? I think we should just remove this line? Having commented css is not that nice.

Reply:
Removed this.